doc/DeveloperGuide.md

```diff
@@ -87,6 +87,29 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### User case: Rename tag
+
+Software System: AddressBook <br>
+Actor: User <br>
+Preconditions: User is logged in. <br>
+Guarantees: The tag will only be changed if the user confirms the changes. <br>
+
+**MSS**
+
+1. User inputs request to rename specific tag.
+2. System prompts user to enter new tag to replace the old one.
+3. User inputs desired new tag.
+4. System prompts user to confirm the new changes.
+5. User confirms the changes to be made.
```

Comment:
Keep in mind that (especially for the project) that the app should have `advanced CLI`.